server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -374,6 +375,21 @@ public boolean equalsAliases(Metadata other) {
      */
     public ImmutableOpenMap<String, MappingMetadata> findMappings(String[] concreteIndices,
```

Comment:
I think we should inline this overload, there's only a couple of other usages of this method in production code and it's a pretty small thing for the callers to pass in a no-op check if that's what they want.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/72234/commits/e2d96baf2c381300d66cbc415623f23dc93f6bc1